tests/testthat/test-enw_filter_reference_dates.R

```diff
@@ -76,8 +75,7 @@ test_that("enw_filter_reference_dates filters as expected when data is present
   )
 })
 
-test_that("enw_filter_reference_dates works with both include and remove days
-          under missing data", {
+test_that("enw_filter_reference_dates works with both include and remove days under missing data", { # nolint line_length_linter.
```

Comment:
```suggestion
test_that("enw_filter_reference_dates works with both include and remove days under missing data", { # nolint: line_length_linter.
```

The colon is needed :grimacing:. The regex is here for reference: https://lintr.r-lib.org/articles/lintr.html#defaults

Reply:
😭